✨Implementa Botão Editar

GuilhermebJunqueira opened the pull request:
Backend:
Adicionado a função de editar as informações dos portais de noticia
ja modificando no banco de dados

Frontend:
Adição do botão editar
Ao pressionar o botão editar aparece um pop-up o qual você pode
editar as informações ja cadastradas dos portais de noticia

GuilhermebJunqueira committed:
✨Implementa Botão Editar

c137santos commented on .idea/.gitignore:
Não pode subir esse arquivo aqui. Inclusive ele está listado no gitignore, por favor, tira do seu pr.

GuilhermebJunqueira committed:
Update .gitignore

GuilhermebJunqueira committed:
Delete .idea/.gitignore

GuilhermebJunqueira committed:
Delete .idea/FATEC-API-3-SEMESTRE.iml

GuilhermebJunqueira committed:
Delete .idea/inspectionProfiles/Project_Default.xml

GuilhermebJunqueira committed:
Delete .idea/misc.xml

GuilhermebJunqueira committed:
Delete .idea/modules.xml

GuilhermebJunqueira committed:
Delete .idea/vcs.xml

GuilhermebJunqueira committed:
Update ApiController.java

GuilhermebJunqueira committed:
Update PortalController.java

GuilhermebJunqueira committed:
Update PortalRepository.java

GuilhermebJunqueira committed:
Update ApiController.java

GuilhermebJunqueira committed:
Update ApiController.java

GuilhermebJunqueira committed:
Update ApisView.vue

GuilhermebJunqueira committed:
Update PortalRepository.java

GuilhermebJunqueira committed:
🐛Correção de funcionamento do codigo

GuilhermebJunqueira committed:
Merge branch '54-editar-portal-de-notcia' of https://github.com/c137santos/FATEC-API-3-SEMESTRE into 54-editar-portal-de-notcia

Commit:
🔀 Merge branch 'main-segunda-sprint' into 54-editar-portal-de-notcia

Commit:
⚡️ Modularizando Modal de edição de Portal Notícia

Realizei a modularização do modal do portal de notícia, assim como ajustei os parametros de adicionar portal ao DB.

Commit:
♻️ Colocando o selectPortal como parametro do v-if

marilia-borgo committed:
🐛 Usa o endpoint correto
